recipe/meta.yaml

```diff
@@ -1,31 +1,31 @@
 {% set name = "fastapi" %}
-{% set version = "0.103.0" %}
+{% set version = "0.112.2" %}
 
 package:
   name: {{ name|lower }}
```

Comment:
We should consider splitting it next time, see https://github.com/fastapi/fastapi/blob/0.112.2/pyproject.toml#L43-L100 because of the existing `fastapi-cli` tool. But conda-forge still has a single output recipe https://github.com/conda-forge/fastapi-feedstock/blob/main/recipe/meta.yaml#L22-L32